.github/workflows/cross-platform.yml

```
@@ -0,0 +1,152 @@
+        with:
+          name: coverage-lcov
+          path: lcov.info
+
+      - name: Upload to Codecov
+        if: matrix.os == 'ubuntu-22.04' && matrix.tls == 'native-tls' && github.repository == 'UncleSp1d3r/gold_digger'
+        uses: codecov/codecov-action@v5
+        with:
+          files: lcov.info
+          flags: unit
+          fail_ci_if_error: false
+          verbose: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)